tests/scripts/thread-cert/Cert_5_5_02_LeaderReboot.py

```diff
@@ -89,6 +96,91 @@ def test(self):
         for addr in addrs:
             self.assertTrue(self.nodes[ROUTER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        ROUTER = pv.vars['ROUTER']
+        MED = pv.vars['MED']
+        leader_pkts = pkts.filter_wpan_src64(LEADER)
+        _rpkts = pkts.filter_wpan_src64(ROUTER)
+
+        # Step 2: The DUT MUST send properly formatted MLE Advertisements
+        _rpkts.filter_mle_cmd(consts.MLE_CHILD_ID_RESPONSE).must_next()
+        _lpkts = leader_pkts.range(_rpkts.index)
+        _lpkts.filter_mle_cmd(
+            consts.MLE_ADVERTISEMENT).must_next().must_verify(lambda p: {0, 11, 9} == set(p.mle.tlv.type))
+
+        _rpkts.filter_mle_cmd(
+            consts.MLE_ADVERTISEMENT).must_next().must_verify(lambda p: {0, 11, 9} == set(p.mle.tlv.type))
+
+        # Step4: Router_1 MUST attempt to reattach to its original partition by
+        # sending MLE Parent Requests to the All-Routers multicast
+        # address (FFxx::xx) with a hop limit of 255.
+        _rpkts.filter_mle_cmd(
+            consts.MLE_PARENT_REQUEST).must_next().must_verify(lambda p: {1, 3, 14, 18} == set(p.mle.tlv.type))
+        lreset_start = _rpkts.index
+
+        # Step 5: Leader MUST NOT respond to the MLE Parent Requests
+        _lpkts.filter_mle_cmd(consts.MLE_PARENT_RESPONSE).must_not_next()
+
+        # Step 6:Router_1 MUST attempt to attach to any other Partition
+        # within range by sending a MLE Parent Request.
+        _rpkts.filter_mle_cmd(
+            consts.MLE_PARENT_REQUEST).must_next().must_verify(lambda p: {1, 3, 14, 18} == set(p.mle.tlv.type))
+        lreset_stop = _rpkts.index
+
+        # Step3: The Leader MUST stop sending MLE advertisements.
```

Comment:
Step 3 between Steps 6 and 7?

Reply:
make sure no MLE_ADVERTISEMENT sent out by leader between step 2 and step 7